### PR TITLE
Autogenerate VPN Tunnel resource

### PR DESCRIPTION
**Pull request by ndmckinley**
These are the changes from Magic Modules in code that was modified by the VPN tunnel PR - should just be some updates to code that was touched by the VPN Tunnel PR.

### CODE REVIEW

File: google/resource_compute_vpn_tunnel.go [hunk 1]
```diff
@@ -211,6 +211,11 @@ func resourceComputeVpnTunnel() *schema.Resource {
 				Optional: true,
 				Elem:     &schema.Schema{Type: schema.TypeString},
 			},
+			"detailed_status": {
+				Type:     schema.TypeString,
+				Optional: true,
```
Comment (danawillow) [inline google/resource_compute_vpn_tunnel.go]: This is settable?

File: google/resource_compute_vpn_tunnel.go [hunk 2]
```diff
@@ -399,6 +406,9 @@ func resourceComputeVpnTunnelRead(d *schema.ResourceData, meta interface{}) erro
 	if err := d.Set("peer_ip", flattenComputeVpnTunnelPeerIp(res["peerIp"])); err != nil {
 		return fmt.Errorf("Error reading VpnTunnel: %s", err)
 	}
+	if err := d.Set("shared_secret", flattenComputeVpnTunnelSharedSecret(res["sharedSecret"])); err != nil {
```
Comment (danawillow) [inline google/resource_compute_vpn_tunnel.go]: I feel like we weren't setting this before on purpose, weren't we?

Comment (ndmckinley) [reply google/resource_compute_vpn_tunnel.go]: Totally - the merge removed the bit that ignored things with ignore_read set.  :(  Fixed and rerunning.

File: google/resource_compute_vpn_tunnel.go (hunk 1 above)
Comment (ndmckinley) [reply google/resource_compute_vpn_tunnel.go]: It's not.

**Comment by ndmckinley** [comment]
All VPN tests still pass.  Submitting.

**Pull request merged**